tsl/src/bgw_policy/move_hyper_inval_api.c

```diff
@@ -0,0 +1,223 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+
+#include <postgres.h>
+
+#include "move_hyper_inval_api.h"
+
+#include "bgw/job.h"
+#include "bgw/job_stat.h"
+#include "bgw/timer.h"
+#include "bgw_policy/job.h"
+#include "bgw_policy/job_api.h"
+#include "guc.h"
+#include "hypertable.h"
+#include "jsonb_utils.h"
+#include "policy_config.h"
+#include <utils/elog.h>
+
+#define DEFAULT_MAX_RUNTIME                                                                        \
+	DatumGetIntervalP(DirectFunctionCall3(interval_in, CStringGetDatum("0"), InvalidOid, -1))
+
+Datum
+policy_move_hyper_inval_proc(PG_FUNCTION_ARGS)
+{
+	if (PG_NARGS() != 2 || PG_ARGISNULL(0) || PG_ARGISNULL(1))
```

Comment:
Can we add test cases for wrong args?

Reply:
There are no tests for other policies since calling these functions with NULL would indicate an internal error, but I added tests for this anyway.